client/client.go

```diff
@@ -59,6 +59,12 @@ func NewClient(cfg *Config) (*Client, error) {
 	if cfg.BaseURL != "" {
 		bu = cfg.BaseURL
 	}
+
+	// ensure base url has trailing forward slash
+	if !strings.HasSuffix(bu, "/") {
+		bu += "/"
+	}
+
```

Comment:
Think this might be duplicating this code a few lines down?

https://github.com/sylabs/scs-library-client/blob/8ea9e5d6ec6823b7dbf03c8e73c66580164acb86/client/client.go#L76-L81